graphql-dgs-spring-boot-micrometer/src/main/kotlin/com/netflix/graphql/dgs/metrics/micrometer/DgsGraphQLMetricsInstrumentation.kt

```diff
@@ -102,6 +112,52 @@ class DgsGraphQLMetricsInstrumentation(
         }
     }
 
+    /**
+     * Port the implementation from MaxQueryComplexityInstrumentation in graphql-java and store the computed complexity
+     * in the MetricsInstrumentationState for access to add tags to metrics.
+     */
+    override fun beginValidation(parameters: InstrumentationValidationParameters): InstrumentationContext<List<ValidationError>> {
```

Comment:
Should we send a PR to the graphql-java folks such that they expose this as part of a utility such that it allows some sort of composability that we can leverage?

Reply:
yeah, that makes sense.